### DISCUSSION TITLE
How do I download the latest version of a template?

### DISCUSSION THREAD

**Discussion by chillikid** [primefaces/community#2863] [PrimeVue]
Hi, 

I purchased the V3 version of a template back in April, and now I want to upgrade to V4, but there seems no obvious way of downloading it - I keep getting taking in a loop to purchase it from scratch again - even though I am logged in and it says I've got free updates till April 2025. 

Thanks

**Comment by j0rgedev** [comment]
Hello @chillikid. Unfortunately, If I'm not mistaken, you'll have to buy another license for v4
